### DISCUSSION TITLE
I use parallel route, normal route does not work

### DISCUSSION THREAD

**Discussion by ErsinCabuk** [vercel/next.js#55895] [Help]
### Summary

I can use Parallel Routes in the layout and they work correctly on my index page (doamin.com/). However, when I try to access a different page of my site, I get a 404 error. In that:
![C0QymYvBAB](https://github.com/vercel/next.js/assets/113672117/b310ce4f-6319-47dd-8d0f-de619bf6ce22)

My folder structure:
![image](https://github.com/vercel/next.js/assets/113672117/48c1c97e-51fd-4a2b-821e-17cfbe6fe732)

My layout code:
![image](https://github.com/vercel/next.js/assets/113672117/e60ce169-e308-4f70-bd90-e89a1e42f2dd)

**Comment by ErsinCabuk** [comment]
I found something, does parallel router only work in Layout?

**Comment by Dragate** [comment]
You need to check the [parallel routes documentation](https://nextjs.org/docs/app/building-your-application/routing/parallel-routes#reload). What's rendered is differentiated by how the navigation is done (client-side or reload). You'll have to use default.tsx to provide a fallback on reload.

If I wanted to hide the `friendBar` and `profile` when reloading the `/shop` route, I'd provide a default.tsx returning null for each of those slots. If I wanted to show the `servers` slot, I'd re-export its page in default.tsx.

**Reply by ErsinCabuk** [reply]
I want servers, friendBar and profile parallel routes to be on each route (page). Do I need to re-export default.tsx for each route?
